src/main/clojure/clara/linter.cljc

```diff
@@ -0,0 +1,28 @@
+(ns clara.linter
+  (:require [clojure.string :as clj-str]))
+
+
```

Comment:
This doesn't yet handle nested conditions - I'd like to handle that as well but it doesn't necessarily need to be in a first pass IMO.